hass_nabucasa/remote.py

```diff
@@ -83,6 +93,7 @@ def __init__(self, cloud: Cloud) -> None:
         self._reconnect_task: asyncio.Task | None = None
         self._acme_task: asyncio.Task | None = None
         self._token: SniTunToken | None = None
+        self._certificate_status: CertificateStatus | None = None
```

Comment:
Instead of `None`, should we have a state saying `NOT_LOADED` ?

Reply:
All statuses outside of `LOADED` would effectively be `NOT_LOADED`, I did not add an initial status for 2 reasons:
- It does not need to be there
- It is similar to the other attributes